Dockerfile

```
@@ -2,7 +2,7 @@ FROM eclipse-temurin:8 AS build
 
 WORKDIR /app
 COPY . .
-RUN ./gradlew --no-daemon --info war -x test
+RUN ./gradlew --no-daemon --info war
```

I think tests should be skipped here, let's do this via CI
```suggestion
RUN ./gradlew --no-daemon --info war -x test
```